docs/topolvm-controller.md

```diff
@@ -76,7 +76,8 @@ metadata:
 spec:
   containers:
   - name: testhttpd
-    image: quay.io/cybozu/testhttpd:0
+    image: ubuntu:20.04
+    command: ["/usr/bin/sleep", "infinity"]
```

Comment:
sleep command does not handle signals properly when it is pid 0.
`registry.k8s.io/pause` provides pause command and it is suitable for this use-case.